engine/xml-config/src/main/java/org/datacleaner/configuration/JaxbConfigurationReader.java

```diff
@@ -559,8 +559,8 @@ private ReferenceDataCatalog createReferenceDataCatalog(ReferenceDataCatalogType
 
                         addVariablePath(name);
 
-                        String filenamePath = getStringVariable("filename", tfdt.getFilename());
-                        String filename = createFilename(filenamePath);
+                        final String filenamePath = getStringVariable("filename", tfdt.getFilename());
+                        final String filename = createFilename(filenamePath);
```

Comment:
Nooooooooooooooooooooooooooooooooooooooooooooooooooo NOT IN HERE! Then I'll need to merge when I do my commit. :scream: :sob: **whine**